src/main/java/com/google/devtools/build/lib/analysis/starlark/StarlarkTransition.java

```diff
@@ -445,6 +446,48 @@ public static BuildOptions addDefaultStarlarkOptions(
     return optionsWithDefaults == null ? fromOptions : optionsWithDefaults.build();
   }
 
+  /*
+   * Removes all build settings explicitly set to their default values from the output build options
```

Comment:
Nit: what precisely does "explicitly" mean here?

i.e. is this equivalent to "removes all build settings set to their default values"? Or is it possible for "implicit" defaults to get in there somehow?